src/search.cpp

```diff
@@ -1250,6 +1250,11 @@ namespace {
       {
           Depth r = reduction<PvNode>(improving, depth, moveCount);
 
+#ifdef ANTI
+          if (pos.is_anti() && pos.can_capture())
+              r -= r ? ONE_PLY : DEPTH_ZERO;
+          else
```

Comment:
I am a bit confused, since I thought this was feedback regarding my changes, but the code you are referring to is unchanged.

I think the `ttCapture` part should never be called in giveaway chess, since there can not be a legal quiet move if a capture is available. I had tried to change the escaping a capture part, but it was not successful so far.